### PR TITLE
PEAR-1663: Developers Guide

### PR DESCRIPTION
**Pull request by craigrbarnes**
## Description
Adds a GDC V2 Application developer's guide to the doc
Moves generated typedoc to API
Adds md-to-pdf
Correct comment in Project Center

Scan with aXe shows no serious issues: If you run in dark mode, there are contrast issues 
## Checklist

- [ ] Added proper unit tests
- [x] Left proper TODO messages for any remaining tasks
- [x] Scanned for web accessibility with **aXe**, and mitigated or documented
      flagged issues

## Screenshots/Screen Recordings (if Appropriate)
image

### CODE REVIEW

**Review by ocshawn — APPROVED** [review]
looks good

**Comment by ocshawn** [comment]
tests are failing may want to run: npm run format

as it looks to be failing on trailing white space

**Comment by craigrbarnes** [comment]
> tests are failing may want to run: npm run format
> 
> as it looks to be failing on trailing white space

Thank you I noticed it and hopefully the last push fixes it.

**Comment by craigrbarnes** [comment]
> > tests are failing may want to run: npm run format
> > as it looks to be failing on trailing white space
> 
> Thank you I noticed it and hopefully the last push fixes it.

It seems to have.

File: docs/DevelopersGuide/Developers_Guide.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# GDC Portal Developer Guide
+# GDC Portal V4 Application Developer Guide
```
Comment (paribartandhakal) [inline docs/DevelopersGuide/Developers_Guide.md:1]: v2?

Comment (craigrbarnes) [reply docs/DevelopersGuide/Developers_Guide.md:1]: Ugh. Thank you!!!

**Pull request merged**